sdlib/d/gc/tcache.d

```diff
@@ -30,14 +61,21 @@ public:
 	}
 
 	void* allocAppendable(size_t size, bool containsPointers) {
-		// Force large allocation rather than slab.
-		enum MinSize = getSizeFromClass(ClassCount.Small);
+		auto asize = alignUp(getAllocSize(size), 2 * Quantum);
```

Comment:
I would have put the alignUp inside, but that probably give the same result in practice.